tests/es/todo_HassListCompleteItem.yaml

```
@@ -0,0 +1,30 @@
+language: es
+tests:
+  - sentences:
+      - elimina manzanas de mi lista de Mercadona
+      - eliminar manzanas en Mercadona
+      - eliminá manzanas de lista Mercadona
+      - borrar manzanas en el listado Mercadona
+      - borra manzanas en el lista del Mercadona
+      - borrá manzanas en la lista Mercadona
+      - elimina manzanas de la lista del Mercadona
+      - quita manzanas en el lista del Mercadona
```

no suena muy natural, no?
```suggestion
      - quita manzanas en el listado del Mercadona
```